src/core/openthread-core-default-config.h

```diff
@@ -1535,4 +1535,34 @@
 #define OPENTHREAD_CONFIG_DISABLE_CCA_ON_LAST_ATTEMPT 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_DIAG_OUTPUT_BUFFER_SIZE
+ *
+ * Define openthread diagnostic mode output buffer size
```

Comment:
nit: openthread -> OpenThread. Also "." at the end of lines (here and the ones below).

Reply:
okay